_episodes/15-transferring-files.md

```diff
@@ -138,6 +86,23 @@ Hit "Quickconnect" to connect! You should see your remote files appear on the ri
 screen. You can drag-and-drop files between the left (local) and right (remote) sides of the screen
 to transfer files.
 
+## Compressing files
```

Comment:
What is the reason for this section here? I am wondering, if this content about compressing files is required or essential for the introduction to HPC.

Reply:
My justification would be that archiving (often involving compression) is a key part of being able to transfer data to/from remote systems. It is really important to build in this good habit for new users as a lot of problems researchers have with HPC are manipulating data between different places in their research workflow. However, re-reading this section I think the heading "Compression" is the wrong one as it is the "Archiving" of multiple files into a single file is *the* key concept we want to get across here. 

I will have a go at rewording/refocussing.

Reply:
I have updated this in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/e0abb4c06b6e0fae4cccba1d3211b8bf9d06f830

Take a look and let me know if you think this is better.

Reply:
Super, if I could buy you a coffee, I'd do so. e0abb4c turns the tide completely and gives it quite some relevance. thanks for putting time into this.